docs/build/messages/read-receipt.mdx

```diff
@@ -195,11 +208,26 @@ if (message.encodedContent.type == ContentTypeReadReceipt) {
 }
 ```
 
-</TabItem></Tabs>
+</TabItem>
+<TabItem value="dart" label="Dart"  attributes={{className: "dart_tab"}}>
+
+Read receipts for Dart haven't been implemented yet.
+
+</TabItem>
+<TabItem value="rn" label="React Native"  attributes={{className: "rn_tab"}}>
+
+Read receipts for React Native haven't been implemented yet.
+
+</TabItem>
+</Tabs>
+
+## Use a read receipt
+
+A read receipt timestamp shouldn't be displayed. Instead, use it to calculate the time since the last message was read. While iterating through messages, you can be sure that the last message was read at the timestamp of the read receipt if the string of the date is lower.
```

Comment:
Above - "While iterating through messages, you can be sure that the last message was read at the timestamp of the read receipt if the string of the date is lower." 

What does this mean? Lower than what? Why the date and not the entire timestamp?